### DISCUSSION TITLE
GUI request

### DISCUSSION THREAD

**Discussion by Undisputed00x** [ALTaleX531/TranslucentFlyouts#126] [Show and tell]
I dont know if this release was intended to do not include any GUI for customization.

**Comment by akaydev-coder** [comment]
It is fully customize using Regedit, Read the Readme, mentioned at the Main Site
/Config/zh-cn.md
The Release is really awesome

**Comment by ALTaleX531** [comment]
I have added so many options in this release that creating GUI is a hard work😂, so I decided to upload it later and have a rest.
But you can still follow the [guidance](https://www.wikihow.com/Use-Regedit) to modify the settings😉

**Comment by dfgwebxqu** [comment]
@ALTaleX531 It is recommended to embed the config markdown in the readme file. If I didn’t read the issues, I would not know the existence of the configuration item...

**Comment by Satanarious** [comment]
Just released the 2nd alpha for [TranslucentFlyoutsConfig](https://github.com/Satanarious/TransparentFlyoutsConfigGUI/releases/tag/v1.0.0-alpha2). Might wanna check it out and lemme know if you encounter any bugs. Tested on TFs [v2.0.0-beta.2](https://github.com/ALTaleX531/TranslucentFlyouts/releases/tag/v2.0.0-beta.2)

**Comment by Satanarious** [comment]
Published the 4th alpha for [TranslucentFlyoutsConfig](https://github.com/Satanarious/TransparentFlyoutsConfig/releases/tag/v1.0.0-alpha.4) today. This release should fix all of the earlier problems and also has a brand new look with just a minor bug from a library I'm using. I'll be working on translation support next. Have a look and let me know if you encounter any bugs. Checkout more on the development [here](https://github.com/Satanarious/TransparentFlyoutsConfig).

**Comment by Satanarious** [comment]
![language_and_description_spread](https://github.com/ALTaleX531/TranslucentFlyouts/assets/42642682/9619bf02-f468-4100-9500-f7d8b7d4a4e7)
Published my first beta release for [TranslucentFlyoutsConfig](https://github.com/Satanarious/TransparentFlyoutsConfig) today. This release include major additions and fixes including:
- Translation Support
- Parameter Description Bottom-Sheet, So you don't have to refer to the config.

Check out the latest release [here](https://github.com/Satanarious/TransparentFlyoutsConfig/releases/latest). Also you can find translation contribution related information [here](https://github.com/Satanarious/TransparentFlyoutsConfig#translation-contribution)

**Comment by Satanarious** [comment]
<table>
<tr>
<td>
<img src="https://github.com/ALTaleX531/TranslucentFlyouts/assets/42642682/4334308b-17eb-48b3-a194-4d05e548c440">
<td>
<img src="https://github.com/ALTaleX531/TranslucentFlyouts/assets/42642682/671e9bc5-630e-4584-adc6-be500ea6d1a8">

</tr>
</table>

Just released v1.2.0 for [TranslucentFlyoutsConfig](https://github.com/Satanarious/TransparentFlyoutsConfig) with several new features including:
- Theming support 
- Internal Functions like Run, Stop, Install, and Uninstall support within the application.
- Automatic Download and Install option which eliminates the need to install TF manually.

Check out the latest release [here](https://github.com/Satanarious/TransparentFlyoutsConfig/releases/tag/v1.2.0).

**Comment by TechySC23** [comment]
@Satanarious and @ALTaleX531 that is from literally last year. Are you serious? There's been no GUI updates for the new features.

**Comment by Satanarious** [comment]
I'm sure that you know that v3.0 for TF was released a couple of days ago. Also, the latest version of TFC is v1.3.3, not this one, because I stopped posting here. Also, there is no documentation for v3.0 for TF yet. This stuff takes time because TF and TFC are two separate software, developed and maintained by two completely different people. There are several hindrances here stifling the development of both the software in unison, since one is dependent on the other. 
You may not see it but there were several changes made between v1.2.0 and 1.3.3 that aren't as apparent as other UI features. Also, I'm working on something else right now. I suggest you give it time. Moreover, AltaleX531 suggested that you do not need the features included in the latest release on TF and the latest version of TFC should work just fine with the old features.
If you are in a hurry, you can always manually edit the registry and get the desired result.

Let me know if you have some other query.

**Comment by TechySC23** [comment]
Sorry but what's going on with that run button? 3.0.0 has the run.bat back again.

**Comment by Satanarious** [comment]
I disabled it in favour of a former TF update that removed the file. You can double-click the run.bat file to use it. I'll bring it back in the next update.

**Comment by TechySC23** [comment]
Thanks

**Comment by Satanarious** [comment]
<table>
<tr>
<img src="https://github.com/Satanarious/TranslucentFlyoutsConfig/assets/42642682/a2f35619-1a15-46c7-9cfa-f7211eaffd62" width=33%>
<img src="https://github.com/Satanarious/TranslucentFlyoutsConfig/assets/42642682/4aee4185-8697-4aba-90c1-b0ef9d65ecfc" width=33%>
<img src="https://github.com/Satanarious/TranslucentFlyoutsConfig/assets/42642682/3beb6d98-feef-4f43-8412-26d7c8982563" width=33%>
</tr>
</table>

TFC [v1.4.0](https://github.com/Satanarious/TranslucentFlyoutsConfig/releases/tag/v1.4.0) with TF v3.0.0+ support is here. Update now to get all the new changes, including:
- TF v3.0.0+ support
- External Functions fix
- Dozens of new Parameters
- Added DisabledList for each category
- Overhaul of the tooltip category
- Added a new Advanced Tab in the settings
- New Animation Tab for DropDown category
- New Reset All button on each page (Left-Click->Default Value | Right-Click->Last Value)

**Comment by TechySC23** [comment]
Yay @Satanarious thanks a lot for the new version! And respect to you as a web developer. I know you took a lot of time because as a dev myself, I know that it takes time to build such amazing things.

**Comment by Satanarious** [comment]
There was a lot of work. And switching between Flutter and Python back and forth was a little tiresome. Nonetheless, couldn't have done it sooner if I didn't hear people were waiting for it. Thanks for the motivation.

**Comment by TechySC23** [comment]
Yup

**Comment by akaydev-coder** [comment]
indeed excellent work @Satanarious ... thought the emoticon reaction is enough. I think people would writedown more when there was a function for comments under the release itself. Open a Issue which is not a Issue but a compliment for the work of the developer does not make sense. 
Anyway Disscusion Tab @ Github is a good malternative for it. :-)

**Comment by Satanarious** [comment]
I know what you're talking about. Makes sense 😁
